test/mpi/runtests.in

```diff
@@ -437,229 +436,230 @@ sub RunList {
             # just skip, do not complain missing unless it is "testlist"
             next;
         }
+        print "Looking in $curdir/$_f\n" if $debug;
```

Comment:
nitpick: there are 2 changes here, right? We are fixing the indentation and also moving a print to after a potential loop iteration skip.

Reply:
Yes, I'll split it.